spectrum/autism_prevalence_map/static/autism_prevalence_map/eslint.config.mjs

```diff
@@ -0,0 +1,12 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+  {files: ["**/*.{js,mjs,cjs,ts}"]},
```

Comment:
Are you trying to lint files that are not created by us? Like files in the node_modules folder?

Reply:
No